plugins/georeplication/rest.go

```diff
@@ -171,11 +174,19 @@ func georepCreateHandler(w http.ResponseWriter, r *http.Request) {
 	// Workaround till {{ volume.id }} added to the marker options table
 	vol.Options["marker.volume-uuid"] = vol.ID.String()
 
+	//save volume information for transaction failure scenario
+	if err := txn.Ctx.Set("oldvolinfo", oldvolinfo); err != nil {
+		logger.WithError(err).Error("failed to set oldvolinfo in transaction context")
```

Comment:
Add WithField("key", "oldvolinfo")

Reply:
TRIVIAL: Also try to use mixedCases, oldvolinfo -> oldVolInfo

Reply:
I think we are not currently not following the convention of using mixedCases in glusterd2, and we should add this rule to the development guide.